README.md

```diff
@@ -370,9 +373,11 @@ airbrake.AddFilter(notice =>
         return null;
 
     // clear environment variables with "token"-related keys
-    foreach (var key in notice.EnvironmentVars.Keys)
-        if (key.Contains("token"))
+    new List<string>(notice.EnvironmentVars.Keys).ForEach(key =>
```

Comment:
I don't see how this is relevant to the NLog integration. If the example is incorrect, it's probably better to open a separate PR.